src/ArmTemplates/Extractor/Models/ExtractorParameters.cs

```diff
@@ -129,6 +131,7 @@ public ExtractorParameters(ExtractorConsoleAppConfiguration extractorConfig)
             this.ExtractSecrets = extractorConfig.ExtractSecrets != null && extractorConfig.ExtractSecrets.Equals("true", StringComparison.OrdinalIgnoreCase);
             this.ExtractIdentityProviders = extractorConfig.ExtractIdentityProviders != null && extractorConfig.ExtractIdentityProviders.Equals("true", StringComparison.OrdinalIgnoreCase);
             this.ExcludeBuildInGroups = extractorConfig.ExcludeBuildInGroups != null && extractorConfig.ExcludeBuildInGroups.Equals("true", StringComparison.OrdinalIgnoreCase);
+            this.ExcludeGlobalPolicy = extractorConfig.ExcludeGlobalPolicy != null && extractorConfig.ExcludeGlobalPolicy.Equals("true", StringComparison.OrdinalIgnoreCase);
```

Comment:
Let's add the possibility of overriding this parameter as well. 
Method: OverrideConfiguration